resources/release/criteria-not-met-template.md

```diff
@@ -1,4 +1,4 @@
-Hi ${RELEASE_OWNER}, </br>
+Hi @${RELEASE_OWNER}, </br>
 
 The below ${TYPE_OF_CRITERIA} criteria for your component has not been met. </br>
```

Comment:
Is it possible that we can tell what exact criteria that's not met here?

Reply:
We have not decided yet. I just added `@` as it was missed.